packages/default/scss/card/_layout.scss

```diff
@@ -0,0 +1,240 @@
+    .k-card-actions {
+        @extend %top-rounded-child;
+        @extend %bottom-rounded-child;
+        padding: $card-actions-padding-y $card-actions-padding-x;
+        border-width: 0;
+        border-style: solid;
+        border-color: inherit;
+        overflow: hidden;
+        flex-shrink: 0;
+        flex-basis: auto;
+
+        .k-card > & {
+            border-top-width: $card-actions-border-width;
+            border-color: inherit;
+        }
+
+        > .k-button.k-flat:first-child {
+            margin-left: (-$button-padding-x);
+        }
+    }
+    .k-card-action {
+        border-width: 0;
+        border-style: solid;
+        border-color: inherit;
+        display: inline-flex;
+        flex: 1 1 auto;
+
+        > .k-button {
+            @include border-radius( 0 );
+            padding: ($card-actions-padding-y + $button-padding-y) $card-actions-padding-x;
+            flex: 1 1 auto;
+        }
+    }
+
+    .k-card-actions-vertical {
+        padding: 0;
+        display: flex;
+        flex-direction: column;
+
+        .k-card-action + .k-card-action {
+            border-top-width: $card-actions-border-width;
+        }
+    }
+    .k-card-actions-stretched {
+        padding: 0;
+        display: flex;
+        flex-direction: row;
+
+        .k-card-action + .k-card-action {
+            border-left-width: $card-actions-border-width;
+        }
+    }
+
+
+    // Deck
+    .k-card-deck {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: nowrap;
+        align-items: stretch;
+        flex: 0 0 auto;
+
+        .k-card {
+            flex: 0 0 auto;
+        }
+        .k-card + .k-card {
+            margin-left: $card-deck-gap;
+        }
+    }
+
+
+    // Group
+    .k-card-group {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: nowrap;
+        align-items: stretch;
+        flex: 0 0 auto;
+
+        .k-card {
+            @include border-radius(0);
+            flex: 0 0 auto;
+
+            > .k-card-header {
+                @include border-radius(0);
+            }
+        }
+        .k-card + .k-card {
+            margin-left: -1px;
+        }
+
+
+        // Border-radius
+        .k-card.k-first {
+            @include border-left-radius( $card-border-radius );
+
+            > .k-card-header {
+                border-top-left-radius: $card-border-radius;
+            }
+        }
+        .k-card.k-last {
+            @include border-right-radius( $card-border-radius );
+
+            > .k-card-header {
+                border-top-right-radius: $card-border-radius;
+            }
+        }
+        .k-card.k-only {
+            @include border-radius( $card-border-radius );
+
+            > .k-card-header {
+                @include border-top-radius( $card-border-radius );
+            }
+        }
+    }
+
+
+}
+
+
+
+@include exports("card/layout/ie11") {
+
+    .k-ie11 {
+
+        .k-card-body {
+            flex: 1 1 auto;
```

Comment:
Perhaps a comment about why this is necessary would be of use here. Perhaps it is a flex bug and a URL can help understanding this, too?

Reply:
I will add a comment. It's an issue with IE 11 and shorthand flex props. We've had it numerous times already...